src/test/java/plugins/XvncPluginTest.java

```diff
@@ -11,21 +11,36 @@
 import org.junit.Test;
 
 import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import org.jenkinsci.test.acceptance.docker.DockerContainerHolder;
+import org.jenkinsci.test.acceptance.docker.fixtures.XvncSlaveContainer;
 import static org.jenkinsci.test.acceptance.plugins.xvnc.XvncJobConfig.*;
+import org.jenkinsci.test.acceptance.po.Slave;
+import org.jenkinsci.test.acceptance.po.WorkflowJob;
+import org.jvnet.hudson.test.Issue;
 
 @WithPlugins("xvnc")
 public class XvncPluginTest extends AbstractJUnitTest {
```

Comment:
This should have `@WithDocker` so it will be skipped if docker is not available.